sklearn/metrics/tests/test_ranking.py

```diff
@@ -427,7 +427,7 @@ def test_auc():
     assert_array_almost_equal(auc(x, y), 0.5)
 
 
-@pytest.mark.filterwarnings("ignore: The 'reorder' parameter")  # 0.22
+# TODO: not sure what to do with this test yet
```

Comment:
As in: leave or remove?

Reply:
As the author of that PR, I think we can remove this test, because we now raise an error when x is neither increasing nor decreasing. I don't think it's a regression since we have a deprecation cycle here (please refer to the parameter description if you want to know the reason to deprecate this parameter).

Reply:
OK I'll keep this in mind when reopening a PR for this deprecation. Thanks.